templates/components/media/media.html.twig

```diff
@@ -18,4 +16,121 @@
 {% set modifier_class = attributes.class %}
 {% set attributes = attributes|without('class') %}
 
-{% extends "@decanter/components/media/media.twig" %}
```

Comment:
This seems like a useful change to the decanter package. Why punt?

Reply:
![shrug](https://media1.tenor.com/images/ce441e4f0e6f115e9eb1b321955c1b80/tenor.gif?itemid=5094560)